CHANGES.md

```diff
@@ -74,6 +82,10 @@ To be released.
  -  Added `BlockChain<T>.GetBalance()` method.  [[#861], [#900]]
  -  Added `Block<T>.TotalDifficulty` property.  [[#666], [#917]]
  -  Added `SwarmOptions` class.  [[#926]]
+ -  Added `Block<T>.ActionsHash` property.  [[#931], [#935]]
+ -  Added `BlockHeader<T>.ActionsHash` property.  [[#931], [#935]]
+ -  Added `Block<T>.PreCommitHash` property.  [[#931], [#935]]
+ -  Added `BlockHeader<T>.PreCommitHash` property.  [[#931], [#935]]
```

Comment:
I guess you did not added `Block<T>.ActionsHash` and `Block<T>.PreCommitHash` property. Did you mean `Block<T>.Hash`, `Block<T>.PreEvaluationHash ` and `Block<T>.EvaluationDigest`?

Reply:
Yes, I need rename about that 😅

Reply:
solved at https://github.com/planetarium/libplanet/pull/935/commits/2fb44dda1dff79829d89cfc497109143d4c6d462